refactor(web): refactor web operator (#729)
instill-ai/pipeline-backend: pkg/component/operator/web/v0/README.mdx

| Field | Field ID | Type | Note |
| :--- | :--- | :--- | :--- |
| Description | `description` | string | The description of the webpage. |
| Source URL | `source-url` | string | The source URL of the webpage. |
| Title | `title` | string | The title of the webpage. |
</div>
</details>

<h4 id="jquery-syntax-examples">jQuery Syntax Examples</h4>

- **Element Selector**: Targets all instances of a specific HTML tag.
  **Example**: `div`
  Extracts all `<div>` elements.

- **ID Selector**: Targets a single element by its unique `id` attribute.
  **Example**: `#header`
  Extracts the element with the `id` of `header`.

- **Class Selector**: Targets all elements with a specific class name.
  **Example**: `.button`
  Extracts all elements with the class `button`.

- **Attribute Selector**: Targets elements based on the presence or value of an attribute.
  **Example**: `[type="text"]`
  Extracts all elements with a `type` attribute equal to `text`.

- **Descendant Selector**: Targets elements that are nested within other elements.
  **Example**: `div p`
  Extracts all `<p>` elements that are inside `<div>` elements.

#### About Dynamic Content

> `TASK_SCRAPE_PAGE` supports to fetch dynamic content from web pages while simulating user behaviors, such as scrolling down. The initial implementation includes the following capabilities:

Reviewer: ```suggestion
`TASK_SCRAPE_PAGE` supports fetching dynamic content from web pages by simulating user behaviours, such as scrolling down. The initial implementation includes the following capabilities:
```
